nav/index.php

```diff
@@ -0,0 +1,50 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" type="text/css" href="/css/cmn.css">
```

Comment:
Could you match the CSS filename to the page filename?  Will be easier to navigate to for later maintenance.